common/src/main/java/org/apache/hertzbeat/common/entity/manager/Tag.java

```diff
@@ -78,7 +78,7 @@ public class Tag {
             accessMode = READ_WRITE)
     @Min(0)
     @Max(3)
-    private byte type;
+    private Byte type;
```

Comment:
Will this change cause null data conversion exceptions?  when type is null in db, but use the `Byte`

Reply:
> Will this change cause null data conversion exceptions? when type is null in db, but use the `Byte`

I think it is ok. `Byte` is compatible with null but `byte` cannot convert null as a primitive data type

Reply:
ok 👍

Reply:
> btw, we need make sure `Byte type` can not null, due here will throws a NullPointerException since null cannot be converted to a primitive value.
> 
> ```
> List<Tag> tags = monitor.getTags().stream().filter(tag -> tag.getType() == (byte) 0).collect(Collectors.toList());
> ```

Okay, i will check it.